english/scream/mom/eatwithmom.md

```diff
@@ -1 +1,3 @@
 I cant eat it alone, lets call mom and share with her.
+
+Mom, however, does not like marshmallows and thinks you are going to spoil your dinner. She also unfortunately does not believe your story about the marshmallows coming alive!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.